projects/core/src/lib/utils/dom/set-element-value.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Sets value to element, and dispatches input event
+ *
+ * @example
+ * maskitoSetElementValue(input, newValue);
+ *
+ * @see {@link https://github.com/taiga-family/maskito/issues/804 issue}
+ *
+ * @return void
+ */
+export function maskitoSetElementValue(
```

Comment:
* https://github.com/taiga-family/maskito/issues/804#issuecomment-1864095197

Reply:
@aktanoff Could you please update this function according to this discussion?

I apologize for the fuss!